app/models/maintenance_tasks/task.rb

```diff
@@ -235,10 +235,10 @@ def after_error(*filter_list, &block)
       #
       # @param exceptions list of exceptions to rescue and report
       # @param report_options [Hash] optionally, supply additional options for `Rails.error.report`.
-      #   By default: <code>{ source: "maintenance_tasks" }</code> or (Rails <v7.1) <code>{ handled: true }</code>.
+      #   By default: <code>{ handled: true, source: "maintenance-tasks" }</code>.
       def report_on(*exceptions, **report_options)
         rescue_from(*exceptions) do |exception|
-          Rails.error.report(exception, source: "maintenance_tasks", **report_options)
+          Rails.error.report(exception, handled: true, source: "maintenance-tasks", **report_options)
```

Comment:
This is the default, let's not pass it, only test it.
```suggestion
          Rails.error.report(exception, source: "maintenance-tasks", **report_options)
```